acs-apiclient/LoginController.cs

```diff
@@ -292,6 +292,7 @@ public void RetrievedAuthCode()
             {
                 loginDelegate.EncounteredError(AcsApiError.InvalidCallbackParameters, "The Auth Code is invalid.");
             }
+            loginDelegate.ProgressChanged("You were authenticated successfully, we're logging you in.");
```

Comment:
@ahmadhusseini should probably be in the strings file to be localized. But i know the message itself is not even confirmed. We should ask @chrisd008 what the wording here should be.